src/display/annotation_layer.js

```diff
@@ -679,6 +682,24 @@ class LinkAnnotationElement extends AnnotationElement {
     link.className = "internalLink";
   }
 
+  /**
+   * Bind attachments to the link element.
+   * @param {Object} link
+   * @param {Object} attachment
+   */
+  _bindAttachment(link, attachment) {
```

Comment:
The code here is more or less the same as in `web/pdf_outline_viewer.js`, is there any possibility to have it in one place ?

Reply:
Similar yes, but not entirely identical. This problem isn't unique to newly added action though, but exists for most of the older methods.
Hence I think that it'd make sense to consider all of these methods together, and not just piecemeal, which is why a follow-up would be more suited in my opinion. However it's not guaranteed that it'd work out all that nicely, given the slight differences, and I'd also really like to avoid making the `PDFLinkService`-class too "unwieldy" in the process (since users may want to create custom ones).